tests/integration/reqs/test_amm_info.py

```
@@ -1,5 +1,9 @@
 from tests.integration.integration_test_case import IntegrationTestCase
-from tests.integration.it_utils import test_async_and_sync
+from tests.integration.it_utils import (
```

Flake8 (7.3.0)

[error] 'tests.integration.it_utils.create_amm_pool_with_mpt' imported but unused

(F401)